src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ControlTests.cs

```diff
@@ -12393,22 +12615,196 @@ public void Control_RescaleConstantsForDpi_Invoke_Nop(int deviceDpiOld, int devi
             control.RescaleConstantsForDpi(deviceDpiOld, deviceDpiNew);
         }
 
+        [Fact]
```

Comment:
```suggestion
        [WinFormsFact]
```

Applies to other new tests